tests/unit/src/test/scala/tests/codeactions/ConvertSingleLineCommentLspSuite.scala

```diff
@@ -0,0 +1,268 @@
+  )
+
+  check(
+    "convert single line comment to block comment if it is defined between method params",
+    """def foo(
+      |  name: String, // another comment
+      |  age: Int, //<<>> important comment
+      |  amount: Int
+      |)
+      |""".stripMargin,
+    ConvertCommentCodeAction.Title,
+    """def foo(
+      |  name: String, // another comment
+      |  age: Int, /* important comment */
+      |  amount: Int
+      |)""".stripMargin,
+    fileName = "script.sc",
+  )
+
+  check(
+    "convert the whole comment cluster expanded in both ways",
+    """val a = 1
+      |// comment start 1
+      |// comment start 2
+      |// <<>>comment middle
+      |// comment end 1
+      |// comment end 2
+      |val b = 2
+      |""".stripMargin,
+    ConvertCommentCodeAction.Title,
+    """val a = 1
+      |/* comment start 1
+      | * comment start 2
+      | * comment middle
+      | * comment end 1
+      | * comment end 2 */
+      |val b = 2
+      |""".stripMargin,
+    fileName = "script.sc",
+  )
+
+  check(
+    "convert the whole comment cluster expanded in both ways -- additional white spaces",
+    """val a = 1
+      |// comment start 1
+      |  // comment start 2
+      |// <<>>comment middle
+      |// comment end 1
+      |  // comment end 2
+      |val b = 2
+      |""".stripMargin,
+    ConvertCommentCodeAction.Title,
+    """val a = 1
+      |/* comment start 1
+      | * comment start 2
+      | * comment middle
+      | * comment end 1
+      | * comment end 2 */
+      |val b = 2
+      |""".stripMargin,
+    fileName = "script.sc",
+  )
+
+  check(
+    "convert the whole comment cluster expanded in both ways -- mixed style comments",
+    """val a = 1
+      |// comment start 1
+      |  /* comment start 2 */
+      |// <<>>comment middle
+      |/* comment end 1 */
+      |  // comment end 2
+      |val b = 2
+      |""".stripMargin,
+    ConvertCommentCodeAction.Title,
+    """val a = 1
+      |/* comment start 1
+      | * comment start 2
+      | * comment middle
+      | * comment end 1
+      | * comment end 2 */
+      |val b = 2
+      |""".stripMargin,
+    fileName = "script.sc",
+  )
+
+  check(
+    "convert the whole comment cluster expanded in both ways -- mixed with other tokens",
+    """val a = 1
+      |// comment start 1
+      |  /* comment start 2 */ val b = 1
+      |// <<>>comment middle
+      |/* comment end 1 */ val c = 2
+      |  // comment end 2
+      |""".stripMargin,
+    ConvertCommentCodeAction.Title,
+    """|val a = 1
+       |// comment start 1
+       |  /* comment start 2 */ val b = 1
+       |/* comment middle
+       | * comment end 1 */ val c = 2
+       |  // comment end 2
+       |""".stripMargin,
+    fileName = "script.sc",
+  )
+
+  check(
+    "show action if line comment is the first line in the file",
+    """// <<>>comment middle""",
+    ConvertCommentCodeAction.Title,
+    """/* comment middle */""",
+    fileName = "script.sc",
+  )
+
+  checkNoAction(
```

Comment:
It seems that this test and the one below are already present earlier in the file (lines 74 and 126). :slightly_smiling_face:

Reply:
Indeed, fixed.